### PR TITLE
feat: add `picocss` example

### PR DESCRIPTION
**Pull request by danielschmitz**
This is a template using the PicoCSS, a minimal CSS Framework for semantic HTML. its clean & beauty like remix <3

### CODE REVIEW

**Review by MichaelDeBoey — CHANGES_REQUESTED** [review]

File: yarn.lock
[the remark this answers is not in the document]
Comment (danielschmitz) [reply yarn.lock:1]: done

Comment (MichaelDeBoey) [inline yarn.lock:1]: Make sure to revert the changes in this file

```sh
git checkout main ./yarn.lock
```

**Review by MichaelDeBoey — CHANGES_REQUESTED** [review]

**Comment by MichaelDeBoey** [comment]
@mcansh Seems like something went wrong with the changes you applied

**Comment by mcansh** [comment]
> @mcansh Seems like something went wrong with the changes you applied

reset the yarn.lock 🥴
